OPAL/br/src/main/scala/org/opalj/br/analyses/cg/EntryPointFinder.scala

```diff
@@ -28,6 +28,13 @@ sealed trait EntryPointFinder {
      * This method must be implemented by any subtype.
      */
     def collectEntryPoints(project: SomeProject): Iterable[Method] = Set.empty[Method]
+
+    /**
+     * Returns ProjectInformationKeys required by this EntryPointFinder
+     * If no extra keys are required, `Nil` can be returned.
+     */
+    /*ABSTRACT*/
+    def requirements(project: SomeProject): ProjectInformationKeys = { Nil }
```

Comment:
Remove the braces. Also, remove the "ABSTRACT" comment. This method is not abstract, it is validly implemented and most entry point finders will not need to override it.